scripts/changelog/change-creator.js

```diff
@@ -24,7 +24,7 @@ function generateRandomIdentifier() {
  * Ref: https://github.com/aws/aws-sdk-js/issues/3691
  */
 function sanitizeFileName(filename) {
-    return filename.replaceAll(/[^a-zA-Z0-9\\.]/g, '-');
+    return filename.replace(/[^a-zA-Z0-9\\.]/g, '-');
```

Comment:
Not relevant?
```suggestion
    return filename.replaceAll(/[^a-zA-Z0-9\\.]/g, '-');
```

This function is only used by developers when creating changelog.

Reply:
Yes, this should not be committed.